waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/account/x5c/X5CAccountStrategy.kt

```diff
@@ -19,7 +19,7 @@ import kotlinx.uuid.generateUUID
 import org.jetbrains.exposed.sql.insert
 import org.jetbrains.exposed.sql.transactions.transaction
 
-object X5CAccountStrategy: PasswordlessAccountStrategy<X5CAccountRequest>() {
+object X5CAccountStrategy : PasswordlessAccountStrategy<X5CAccountRequest>() {
```

Comment:
this also feels like a change coming from https://github.com/walt-id/waltid-identity/pull/746